packages/core/src/tests/workflow-steps/workflow-steps.e2e.ts

```diff
@@ -37,6 +37,36 @@ regressionTest.describe('workflow-steps', () => {
       stepItem.disabled = true;
     });
 
+    const workflowSteps2 = page.locator('#workflow-steps-2');
+    const stepItemLocator1 = workflowSteps2.locator('ix-workflow-step').nth(0);
+    await stepItemLocator1.evaluate((stepItem: any) => {
+      stepItem.status = 'error';
+    });
+
+    const status = await stepItemLocator1.evaluate(
+      (stepItem: any) => stepItem.status
+    );
+    expect(status).toBe('error');
+
+    await stepItemLocator1.evaluate((stepItem: any) => {
+      stepItem.status = 'open';
+    });
+
+    const workflowSteps3 = page.locator('#workflow-steps-3');
+    const stepItemLocator2 = workflowSteps3.locator('ix-workflow-step').nth(0);
+    await stepItemLocator2.evaluate((stepItem: any) => {
+      stepItem.status = 'open';
+    });
+
+    const status2 = await stepItemLocator2.evaluate(
+      (stepItem: any) => stepItem.status
+    );
+    expect(status2).toBe('open');
+
+    await stepItemLocator2.evaluate((stepItem: any) => {
+      stepItem.status = 'done';
+    });
+
```

Comment:
This should not go into the 'disable' test case. Should be covered by basic test already.